app/src/main/res/layout/layout_unlock.xml

```diff
@@ -18,21 +18,21 @@
             <LinearLayout
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:layout_marginTop="56dp"
+                android:layout_marginTop="32dp"
                 android:gravity="center_horizontal"
                 android:orientation="vertical">
 
                 <ImageView
                     android:id="@+id/setup_fingerprint"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
+                    android:layout_width="75dp"
```

Comment:
Why not wrap?

Reply:
We could use different assets. Would be a minor performance improvement. I am fine either way